docs/docs/migration-guide.mdx

```diff
@@ -10,6 +10,15 @@ description: |
 This page contains information about changes between major versions and
 how you can migrate from one version to another.
 
+## Rasa 2.1 to Rasa 2.2
+
+### Policies
+
+[Policies](./policies.mdx) now require a `**kwargs` argument in their constructor and `load` method.
```

Comment:
we also have a changelog for this, right?

Reply:
hmm I don't think so. Will we have one changelog for the whole of continuous training?

Reply:
we should also add a changelog mentioning this change and then e.g. point to the migration guide.

> Will we have one changelog for the whole of continuous training?

I think this change is somewhat unrelated to the training (we needed it, but we could have also done the policy interface change on its own) so I think it deserves it's own changelog. Especially because it's a deprecation.

Reply:
ok I can add it in this PR then. What issue number should we use?

Reply:
mhm, either the number of this PR or probably better the one from the PR which introduced the change.